core/domain/services/admin/events/editor/FormBuilder.php

```diff
@@ -26,14 +28,16 @@ class FormBuilder implements EventEditorDataInterface
     /**
      * @param int $eventId
      * @return array
+     * @throws EE_Error
+     * @throws ReflectionException
      */
     public function getData(int $eventId): array
     {
-        /** @var EE_Event */
+        /** @var EE_Event $event */
```

Comment:
inline phpdocs need to be the same format as those for a function,
ie: they need to state the type AND the variable name

Reply:
Thanks